alacritty_terminal/src/term/color.rs

```diff
@@ -290,19 +263,19 @@ mod tests {
 
     #[test]
     fn contrast() {
-        let rgb1 = Rgb { r: 0xff, g: 0xff, b: 0xff };
-        let rgb2 = Rgb { r: 0x00, g: 0x00, b: 0x00 };
-        assert!((rgb1.contrast(rgb2) - 21.).abs() < f64::EPSILON);
+        let rgb1 = Rgb::new(0xff, 0xff, 0xff);
+        let rgb2 = Rgb::new(0x00, 0x00, 0x00);
+        assert!(((*rgb1).contrast(*rgb2) - 21.).abs() < f64::EPSILON);
 
-        let rgb1 = Rgb { r: 0xff, g: 0xff, b: 0xff };
-        assert!((rgb1.contrast(rgb1) - 1.).abs() < f64::EPSILON);
+        let rgb1 = Rgb::new(0xff, 0xff, 0xff);
+        assert!(((*rgb1).contrast(*rgb1) - 1.).abs() < f64::EPSILON);
 
-        let rgb1 = Rgb { r: 0xff, g: 0x00, b: 0xff };
-        let rgb2 = Rgb { r: 0x00, g: 0xff, b: 0x00 };
-        assert!((rgb1.contrast(rgb2) - 2.285_543_608_124_253_3).abs() < f64::EPSILON);
+        let rgb1 = Rgb::new(0xff, 0x00, 0xff);
+        let rgb2 = Rgb::new(0x00, 0xff, 0x00);
+        assert!(((*rgb1).contrast(*rgb2) - 2.285_543_608_124_253_3).abs() < f64::EPSILON);
 
-        let rgb1 = Rgb { r: 0x12, g: 0x34, b: 0x56 };
-        let rgb2 = Rgb { r: 0xfe, g: 0xdc, b: 0xba };
-        assert!((rgb1.contrast(rgb2) - 9.786_558_997_257_74).abs() < f64::EPSILON);
+        let rgb1 = Rgb::new(0x12, 0x34, 0x56);
+        let rgb2 = Rgb::new(0xfe, 0xdc, 0xba);
+        assert!(((*rgb1).contrast(*rgb2) - 9.786_558_997_257_74).abs() < f64::EPSILON);
     }
```

Comment:
These tests really should have been moved to VTE, mind creating a new PR for that?

Reply:
https://github.com/alacritty/vte/pull/87

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/d067434b45ddb26b5dd181a98d6fc022962a9a51.